ang/civicase/case/details/linked-cases-tab/directives/case-details-linked-cases-tab.directive.js

```diff
@@ -0,0 +1,39 @@
+(function (angular, loadCrmForm, getCrmUrl) {
```

Comment:
I think `loadCrmForm, getCrmUrl` is not being used, please check.

Reply:
Yes, I created this directive. These functions are being used here:

https://github.com/compucorp/uk.co.compucorp.civicase/pull/467/files#diff-3a285c51bfed1e978fe91c81d6e97996R33